actions/app.go

```diff
@@ -44,13 +47,19 @@ func App() *buffalo.App {
 	mem := memory.New()
 	storageReader := mem
 	storageWriter := mem
+	mgoStore := mongo.NewMongoUserStore("127.0.0.1:27017")
```

Comment:
configurable?

Reply:
eventually but I'd like to add that in a later PR - setting up all the different storage options